tests/de.uka.ipd.sdq.pcm.stochasticexpressions.tests/src/de/uka/ipd/sdq/pcm/stochasticexpressions/tests/CharacterisedVariableParseResultPostProcessorTest.java

```diff
@@ -52,13 +53,15 @@ public void testChangedRootObject() {
         assertEquals(input.hasSyntaxErrors(), actual.hasSyntaxErrors());
         assertThat(actual.getRootASTElement(), is(instanceOf(CharacterisedVariable.class)));
         var newVariable = (CharacterisedVariable) actual.getRootASTElement();
+        assertThat(newVariable.getId_Variable(), is(instanceOf(VariableReference.class)));
+        assertEquals("test", newVariable.getId_Variable().getReferenceName());
         assertEquals(VariableCharacterisationType.TYPE, newVariable.getCharacterisationType());
     }
 
     @Test
     public void testChangedNestedObject() {
-        var variable1 = createVariable("VALUE");
-        var variable2 = createVariable("test");
+        var variable1 = createVariable("test", "VALUE");
+        var variable2 = createVariable("test", "bla");
```

Comment:
I'd suggest something other than "bla", for example "foo"

Reply:
You are right, changed that.